actions/publish-github-pages/action.yml

```diff
@@ -1,10 +1,45 @@
 name: 'Publish GitHub Pages'
-description: 'Publish the Hugo-based CloudPosse docs website to GitHub Pages.'
+description: 'Publish the Hugo-based Cloud Posse docs website to GitHub Pages.'
+inputs:
+  CONTENT:
```

Comment:
I believe the convention is to lower-case these. They will still be upper case envs.